tests/src/main/java/org/everit/audit/ri/ecm/tests/AuditComponentTest.java

```diff
@@ -169,11 +169,11 @@ private void assertAuditApplicationExists(final String expectedApplicationName)
 
       QApplication qApplication = QApplication.application;
 
-      return new SQLQuery(connection, configuration)
+      return new SQLQuery<String>(connection, configuration)
+          .select(qApplication.applicationName)
           .from(qApplication)
           .where(qApplication.applicationName.eq(expectedApplicationName))
-          .uniqueResult(qApplication.applicationName);
-
+          .fetchFirst();
```

Comment:
The alternative function of uniqueResult is fetchOne, not fetchFirst. The fetchFirst is the alternative of singleResult that does not throw exception in case there are multiple results.